CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Minor: Added `in:<channels>` search filter to find messages sent in specific channels. (#2299, #2634)
```

Comment:
Only issue I could potentially see with the way this is phrased, is a possible confusion with [Filters](https://wiki.chatterino.com/Filters/). We could consider changing it as follows:
```suggestion
- Minor: Added `in:<channels>` search predicate to find messages sent in specific channels. (#2299, #2634)
```
Looking for additional input though, as the phrase "search predicate" is not really communicated to the user anywhere in the app. I have opened https://github.com/Chatterino/wiki/issues/58 to address this.

Reply:
Agreed about the possible confusion, the only reason I didn't write predicate is because I realized users are currently unaware of the term. I can wait for a decision on how to word this.

Reply:
I'm fine with keeping the term 'filter' in public terminology, considering we're basing this feature heavily on Discord's search feature, they seem to have originally called them 'search options' but now refer to them as filters https://support.discord.com/hc/en-us/articles/115000468588